src/components/App/MenuContainer/NavigationMenuList/NavigationMenuListStyles.ts

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+import { Props } from './NavigationMenuList';
+
+export const StyledNavMenu = styled.ul`
+  margin: ${(props: Props) => (props.addNestedMenu ? '40px' : '93px')} 0 0 0;
```

Comment:
If yesterday you said me about this, in my view create wrap function much better

Reply:
it is okay, bro. This method is more compact.